docs/product/guidelines/theming-new.html

```diff
@@ -0,0 +1,537 @@
+---
+layout: page
+title: Theming
+description: Stacks provides a robust theming API to handle theming in various contexts.
+---
+
+<!-- TODO: include new theme variables -->
+
+<section class="stacks-section">
+    {% header "h2", "Light, dark, and high contrast modes" %}
+    <p class="stacks-copy">Stacks supports light, dark, and high contrast modes. By using Stacks, you will get each of these modes for free. By using a Stacks component, an atomic color class, or CSS variable directly, your theme will switch appropriately based on the following methods: </p>
+    <ol class="stacks-list">
+        <li>You can apply <code class="stacks-code">.theme-system</code> to the <code class="stacks-code">body</code> element. This will change colors based on the <code class="stacks-code">prefers-color-scheme</code> media query, which is ultimately powered by the user’s system or browser settings. This can be preferable for folks who have their system turn to dark mode based on ambient light or time of day.</li>
+        <li>Alternatively, you can set a dark mode that is not system dependent by attaching <code class="stacks-code">.theme-dark</code> to the <code class="stacks-code">body</code> element.</li>
+        <li>Adding <code class="stacks-code">.theme-highcontrast</code> to the <code class="stacks-code">body</code> element will boost colors to WCAG Level AAA contrast ratios in as many places as possible. This mode stacks on top of both light and dark modes. The only exception is branded themed colors remain untouched by high contrast mode.</li>
+    </ol>
+    <p class="stacks-copy">There are also <a class="s-link" href="{{ "/product/guidelines/conditional-classes#dark-mode" | url }}">conditional classes</a> that can be applied to override assumed dark mode colors, force light mode, or to force dark mode. Forcing modes can be good for previews in admin-only situations.</p>
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Theming API" %}
+    <p class="stacks-copy">Stacks provides an API to apply custom themes.</p>
+
+    {% header "h3", "Programmatic theme generation" %}
+    <p class="stacks-copy">Use <code class="stacks-code">.create-custom-theme-hsl-rgb-variables(@color, @tier, @modeCustom)</code> to create a custom theme.</p>
```

Comment:
Can I suggest to create a list of tradeoffs we are making for reducing liability / maintaining parity?
Nothing fancy - adding something to [this document](https://docs.google.com/document/d/18JkYEVioPhPmHGZaV-ba8S-3rl_zdbMIhpqtsX7iqYk/edit?usp=sharing) should be enough.
I am suggesting this because we could reflect on those later this month when we are close to release 2.0 and decide if it's worth adjusting them as part of this major bump or if we are ok to accept them as "tech debt" for the foreseeable future.

So my suggestion is to make a note about the ideal API we would like and leave things as they are for this PR. Potentially revisit later when we are a bit closer to releasing 2.0.
How does it sound?